### ISSUE TITLE
Full Site Editing

### ISSUE THREAD

**Issue by kwoodall** [tomusborne/generatepress#427]
### Description
Now that this is standard in WordPress 6+ let's get FSE implemented in GP!

---

**Comment by eliquame** [comment]
Please no! Never!

They should further develop the generateblocks plugin (free version) and give more freedom (visual settings) in Generatepress for the flex and grid settings.

**Comment by SchneiderSam** [comment]
> Now that this is standard in WordPress 6+ let's get FSE implemented in GP!

Yes, I agree with that. If GP wants to have a future in Wordpress, there is no way around FSE. 

> Please no! Never!

Why not? FSE is really fun. What GP does with elements is actually FSE. Only that FSE is a tad clearer. And it belongs to the core. Sooner or later GP has to bring a FSE solution. Related: https://github.com/tomusborne/generatepress/issues/389

**Comment by SchneiderSam** [comment]
Spectra has released a FSE theme a few days ago: https://wordpress.org/themes/spectra-one/

 We already know the argument that GP is already a FSE theme, but that's not really the case. Are there any plans that we can use GP in FSE mode soon?

**Comment by SchneiderSam** [comment]
I know, I know: surely you all have this on your radar. But I think this post alone is so powerful that as a loyal GP user, I want nothing more than for Tom and his team to be a part of this awesome WordPress future: https://make.wordpress.org/design/2023/08/10/admin-design-kickoff/

You have already explained the FSE topic very intensively and publicly. Nevertheless, I hope that the idea is not completely thrown overboard and one day Tom and his team will develop a young brother next to GP.... :)

**Comment by SchneiderSam** [comment]
@Calvin-Susan and @tomusborne Oh man, what a wonderful interview. https://www.youtube.com/live/71HT6_fWhLY?si=5UC2TaDWUgG8loms

I ended up watching it in the evening (instead of Amazon Prime)... 😅

The last interview (2023?!) was amazing too, and I thoroughly enjoyed it. Getting those insights was absolutely mind-blowing (even little things like "Why Global Styles and not Global Classes").

Your plans to work on Woo are also fantastic. THANK YOU!

However, I must admit I was really looking forward to the FSE topic. My team keeps asking me: "Yes, but when do we use the Customizer, and when do we use GB?" As Kyle mentioned in the interview, GB is gradually taking over more areas from GP. The only things we still use GP for are global colors, fonts, some general settings here and there, and the menu.

This is where I find FSE particularly intriguing: it all feels more native. GP somehow feels "old." It does what it needs to do, and it does it very well. For me personally, the biggest FSE game-changer is visually editing elements, templates in the entire context, and moving along with the WordPress core.

Oh, and speaking of the menu: it feels VERY odd in GP. I have such a hard time with it (and I've been using GP and GB for over 5-6 years). Being able to style the menu completely in the Gutenberg editor would be a delight. And that's where FSE comes into play again, isn't it?

I get it: it's very complex, and Tom's arguments are TOTALLY understandable. It's just that more and more exciting and good FSE themes are hitting the market, which are very intriguing and simply follow the core path. A very recent example is https://rockbase.co/ - clearly, it's still in its infancy compared to GP. The elements alone make a huge difference in GP. But everything else: Global Colors, fonts, FSE UI are just nice.

**Comment by diggeddy** [comment]
Closing. FSE is a block theme feature. GP has already set its own full site editing features with GP elements. More improvements to come
